.github/workflows/release.yml

```diff
@@ -81,15 +81,23 @@ jobs:
 
       - name: "Build binaries"
         timeout-minutes: 60
-        run: "cargo build --release -p dora-cli"
+        run: "cargo build --release -p dora-coordinator -p dora-cli -p dora-daemon"
```

Comment:
Why are we building the `dora-coordinator` and `dora-daemon` binaries again? I thought that we want to integrate them into the CLI binary?

(same for the other jobs)

Reply:
Typo sorry.